E3db/Classes/IdentityAPI.swift

```diff
@@ -122,17 +122,31 @@ public struct NoteCredentials {
     let signingKeyPair: SigningKeyPair
 }
 
+public struct PublicRealmInfo: Codable {
+    let name: String
+    let brokerId: UUID?
+    let domain: String
+    
+    enum CodingKeys: String, CodingKey {
+        case name
+        case brokerId = "broker_id"
+        case domain
+    }
+}
+
 public struct PasswordNoteData: Codable {
     struct idConfig: Codable {
         let realmName: String
+        let realmDomain: String?
         let appName: String
         let apiUrl: String
-        let username: String
+        let username: String?
```

Comment:
regarding the conversation at standup:
swift ignores any extra fields in the json, but will fail if one of the expected fields isn't included. i made some of the non-essential fields optional so that it will not fail unnecessarily.

Reply:
This lack of username is something that @Mrhea is also seeing. Is this field just not provided by the backend anymore?

Reply:
@efabens @Mrhea I think some SDKs include the username in the note that's created during registration of an identity & some don't. If I register an identity with swift, it has a username as part of the config, but if I register with the js-sdk, it doesn't have a username. The java sdk must be similar to swift.